### ISSUE TITLE
Illinois April 7 Data is wrong

### ISSUE THREAD

**Issue by jason-patel-ensighten** [CSSEGISandData/COVID-19#2060]
Illinois reported 1317 cases on 4/7 but the dataset seems to be updated to only show 9 additional cases, so this seems to be in error?

**Comment by netbymatt** [comment]
13,549 is the total number of confirmed cases for Illinois on 4/7/2020 per the Illinois Department of Public Health.

**Comment by jason-patel-ensighten** [comment]
Correct, 1317 of those cases occurred on 4/7, in the dataset it shows only 9 have happened on 4/7

**Comment by alfkoehn** [comment]
Do you guys happen to know if the number of deaths is correct?

**Comment by netbymatt** [comment]
Deaths are incorrect and should be 370.

**Comment by tchelle** [comment]
It looks like the Illinois numbers for 4/7 rolled into 4/8. It would be great to have this corrected since the daily trends are the most useful analysis at the moment

**Comment by agentdanger** [comment]
Confirming that this is still an issue, and was repeated on 4/9 with no new deaths reported.  That's not correct.  https://coronavirus.illinois.gov/s/

**Comment by mndrake** [comment]
This data is the basis for the [IMHE projections](https://covid19.healthdata.org/united-states-of-america/illinois) that is now understating the peak to do no "new deaths" for a few days for Illinois. Since this study is reported on [here](https://newschannel20.com/news/local/study-projects-covid-19-deaths-in-illinois-to-peak-in-mid-april) and [here](https://wgntv.com/news/coronavirus/new-projection-illinois-hospitals-will-not-be-overrun-with-covid-19-patients/) this is giving the false hope that the peak will be here earlier with far fewer deaths than it probably will be.  

The historic data of deaths and confirmed cases for IL can be found at (https://www.dph.illinois.gov/sitefiles/COVIDHistoricalTestResults.json). Can Illinois be removed going forward if the data will not be corrected. Chicago is one of the hot spots in the US and it is borderline irresponsible IMHO to reporting erroneous data to this magnitude.
